install_scripts/templates/kubernetes/init.sh

```diff
@@ -946,15 +946,19 @@ if [ "$AIRGAP" = "1" ]; then
 fi
 
 kubernetesDeploy
+
 installCliFile \
     "kubectl exec -c replicated" \
     '$(kubectl get pods -o=jsonpath="{.items[0].metadata.name}" -l tier=master) --'
+logSuccess "Installed replicated cli executable"
+
+installAliasFile
```

Comment:
added another issue for this in 2.35
https://app.clubhouse.io/replicated/story/19400/replicated-alias-should-work-on-all-masters